CAP/gap/MethodRecord.gi

```diff
@@ -451,6 +451,41 @@ PreCompose := rec(
   compatible_with_congruence_of_morphisms := true,
 ),
 
+SumOfMorphisms := rec(
+  filter_list := [ "category", "object", "list_of_morphisms", "object" ],
+  io_type := [ [ "source", "morphisms", "range" ], [ "source", "range" ] ],
```

Comment:
```suggestion
```
When giving `input_arguments_names`, `output_source_getter_string` and `output_range_getter_string`, `io_type` is not required anymore.